Automattic/jetpack
# External Media: Update Google Photos connect

Clarifies the Google Photos context as requested in their [UX guidelines](https://developers.google.com/photos/library/guides/ux-guidelines).




#### Changes proposed in this Pull Request:

* Update body and button text in the Google Photos connect modal.

#### Jetpack product discussion


See p4hfux-5P7-p2#comment-8601

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Go to the Editor and insert an image block.
* Click Select image
* Click Google Photos
* The changes should be visible in the modal that opened.
